CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# CONTRIBUTING GUIDELINES
+
+[日本語のコントリビューティングガイドライン](CONTRIBUTING_ja.md)
+
+## About of decomoji add or remove
+
+It does not accept in principle.
+
+Use decomoji-template.psd if you want to use new one.
```

Comment:
You can use `decomoji-template.psd` if you want to add a new decomoji you created.